lib/teamcity.js

```diff
@@ -116,6 +116,20 @@ function Teamcity(runner, options) {
 				log(formatString(TEST_FAILED, test.title, err.message, err.stack, flowId));
 			}
 		}
+		// Log testFinished for failed hook (hook end event is not fired for failed hook)
+		if (recordHookFailures && !test.title.includes(ignoreHookWithName)) {
```

Comment:
Hey @DJ-Glock can you double check this as I didn't pick this up in the previous PR but if you have a test title that includes the string undefined its going to evaluate to true because ignoreHookWithName is undefined when not set. 

Can you initialize this to false instead like the other reporter options?

e.g
`'testing for undefined'.includes(undefined)` //returns true

Reply:
Might need a more comprehensive check than defaulting to false as includes also returns true on a false

`'testing for false'.includes(false)`

Reply:
Nice catch!